src/java/test/org/apache/zookeeper/server/persistence/FileTxnSnapLogTest.java

```diff
@@ -159,4 +159,222 @@ public void onTxnLoaded(TxnHeader hdr, Record rec) {
             }
         }
     }
+
+    @Test
```

Comment:
there is a lot of code duplication in these tests, i'm wondering if they can be cleaned up.

Reply:
@afine I've improved/shortened the test code